coinstacks/ethereum/api/src/client.ts

```diff
@@ -0,0 +1,25 @@
+import WebSocket from 'ws'
```

Comment:
do we actually wanna commit this file?

Reply:
went back and forth, useful for when we mock client package, can remove it then I was thinking?